pkg/codegen/go/gen_program_test.go

```diff
@@ -112,7 +116,9 @@ func newTestGenerator(t *testing.T, testFile string) *generator {
 		}
 
 		g := &generator{
-			program: program,
+			program:            program,
+			jsonTempSpiller:    &jsonSpiller{count: 0},
+			ternaryTempSpiller: &tempSpiller{count: 0},
```

Comment:
Nit: `count` does not need to be zero-initialized; it will automatically be set to the zero value.